kythe/cxx/indexer/cxx/IndexerASTHooks.cc

```diff
@@ -1943,6 +1943,29 @@ bool IndexerASTVisitor::VisitInitListExpr(const clang::InitListExpr* ILE) {
   return true;
 }
 
+bool IndexerASTVisitor::TraverseBinAssign(clang::BinaryOperator* BO) {
+  if (auto rhs = BO->getRHS(), lhs = BO->getLHS();
+      lhs != nullptr && rhs != nullptr) {
+    if (!WalkUpFromBinAssign(BO)) return false;
+    if (!TraverseStmt(lhs)) return false;
+    auto scope_guard = PushScope(Job->InfluenceSets,
+                                 absl::flat_hash_set<const clang::Decl*>());
```

Comment:
I have a trivial PR to make this work as `PushScope(.., {});` if you'd like to wait (but my workstation Bazel was just updated from underneath me so I'll have to do that first).

Reply:
Sure, that'd make this a lot nicer to read.